cli/flox/src/commands/activate.rs

```diff
@@ -331,11 +332,7 @@ impl Activate {
             .unwrap_or(utils::colors::INDIGO_300.to_ansi256().to_string());
 
         let mut exports = HashMap::from([
-            // TODO: honor `--mode` flag
-            (
-                FLOX_ENV_VAR,
-                rendered_env_path.development.to_string_lossy().to_string(),
-            ),
+            (FLOX_ENV_VAR, mode_link_path.to_string_lossy().to_string()),
```

Comment:
suggestion: we also need to pass `--mode` to the activation script

Reply:
The location of this comment is a little confusing. Are you suggesting we pass it as an environment variable like this one, or pass it through as an argument to the other activation functions?

Reply:
Done